backend/cmd/headlamp_test.go

```diff
@@ -492,7 +496,7 @@ func TestDrainAndCordonNode(t *testing.T) {
 		}
 
 		rr, err = getResponse(tc.handler, "GET",
-			fmt.Sprintf("/drain-node-status?cluster=%s&nodeName=%s", drainNodePayload.Cluster, drainNodePayload.NodeName), nil)
+			fmt.Sprintf("/drain-node-status?cluster=%s&nodeName=%s", drainNodePayload.Cluster, drainNodePayload.NodeName), nil) //nolint:lll
```

Comment:
For each time we disable a linter/type checker it should have an explanation of why it was disabled as a comment. Or it should be fixed.

Reply:
Fixed

Reply:
I still see `//nolint:lll` without an explanation.

Reply:
Done now